.github/scripts/check_binary_fixture_size.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Check if a directory is provided as an argument
+if [ $# -eq 0 ]; then
+  echo "Usage: $0 <directory>"
+  exit 1
+fi
+
+directory="$1"
+
+# Check if the directory exists
+if [ ! -d "$directory" ]; then
+  echo "Directory not found: $directory"
+  exit 1
+fi
+
+# Use find to locate all files in the directory and its subdirectories
+found_large_files=0
+while IFS= read -r -d '' file; do
+  # Check if the file size is greater than 100 bytes
+  if [ $(wc -c < "$file") -gt 100 ]; then
```

Comment:
might as well make 100 a variable at the top... also I realized after our discussion that it's going to be larger than 100 almost all of the time since we're capturing extra metadata too (maybe block at 600?)